### PR TITLE
Fix/demo vue

### PR DESCRIPTION
**Pull request by hieu-w**
This PR is about:

- [x] Upgrade use @toruslabs/vue-components, @toruslabs/vue-icons package
- [x] Update to new <script setup> tag in vue3
- [x] Update to new W3A design system
- [x] Fix minor bugs
- [x] Migrate from vue-cli to vite

### CODE REVIEW

File: examples/vue-example/vite.config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import vue from "@vitejs/plugin-vue";
+import { defineConfig } from "vite";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+    alias: {
+      assert: "empty-module",
```
Comment (chaitanyapotti) [inline examples/vue-example/vite.config.js:10]: i don't think you need to define aliases for vite anymore

File: examples/vue-example/package.json [hunk 2]
```diff
@@ -16,32 +17,32 @@
...
     "@web3auth/ethereum-provider": "^7.1.2",
     "bn.js": "^5.2.1",
     "bs58": "^5.0.0",
-    "core-js": "^3.33.3",
     "elliptic": "^6.5.4",
     "ethereum-cryptography": "^2.1.2",
     "ethers": "^6.8.1",
     "openlogin": "file:../../packages/wrapper/",
-    "vue": "^3.3.8"
+    "vite": "^5.3.3",
```
Comment (chaitanyapotti) [inline examples/vue-example/package.json:31]: move vite and plugin vue to devdeps

Comment (hieu-w) [reply examples/vue-example/package.json]: Updated

File: examples/vue-example/vite.config.js (hunk 1 above)
Comment (hieu-w) [reply examples/vue-example/vite.config.js]: Removed

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**